### DISCUSSION TITLE
How to extract the CSS generated by preset-icons into a separate CSS file?

### DISCUSSION THREAD

**Discussion by euwei7** [unocss/unocss#3307] [Q&A] [answered]
Currently, the styles of the icons are mixed together with the regular CSS styles, resulting in a bloated CSS file. I would like to separate these icon styles from the output file and create a standalone file called "icons.css". This way, I can leverage CDN to accelerate the loading speed of the icons.css file.

**Comment by zyyv** [comment, marked as answer]
https://unocss.dev/config/layers#layers

Icons preset generated css in a single layer, try to use `import uno:icons.css` in someway.
